sound/soc/sof/sof-priv.h

```diff
@@ -407,6 +407,8 @@ struct snd_sof_dev {
 	int dma_trace_pages;
 	wait_queue_head_t trace_sleep;
 	u32 host_offset;
+	/* FIXME: combine dtrace flags into one enum */
+	bool dtrace_eof;
```

Comment:
we have too many DMA trace related in this root snd_sof_dev structure, and we actually need more comments to describe how they work, worthy to refine them especially when we plan to provide disabling DMA trace option, but we can do it next.

Reply:
Ack, agreed this would be good to do.